deployment/roles/metron_ui/tasks/main.yml

```diff
@@ -23,46 +23,33 @@
       - { package: 'nodejs'}
       - { package: 'npm'}
 
-- name: Download nvm installer
-  get_url: url=https://raw.githubusercontent.com/creationix/nvm/v0.30.1/install.sh dest=/tmp/nvm-install.sh
-
-- name: Change nvm installer permissions
-  file: path=/tmp/nvm-install.sh mode=0755
-
-- name: Install nvm
-  shell: /tmp/nvm-install.sh creates=~/.nvm/nvm.sh
-
-- name: Install Metron UI
-  shell: source ~/.bashrc && nvm install 0.10.35 && nvm use 0.10.35 && nvm alias default 0.10.35 && npm install -g opensoc-ui creates=/root/.nvm/v0.10.35/lib/node_modules/opensoc-ui/package.json
-
-- name: Add Metron UI config
+- name: Copy Metron UI source
```

Comment:
I am deploying Metron with your PR out to AWS/EC2.  One thing I have noticed so far is that this recursive copy takes FOREVER.  Sorry, I don't have a specific duration to quote, but it seems excessive.

One thing I remember reading is that the 'copy' module in Ansible does not scale well to lots of files.  We may need to tarball up the code and copy it over that way.  Or perhaps you can think of a better alternative.

http://docs.ansible.com/ansible/copy_module.html

> Note
> The “copy” module recursively copy facility does not scale to lots (>hundreds) of files. For alternative, see synchronize module, which is a wrapper around rsync.